courses/management/utils.py

```diff
@@ -147,6 +148,10 @@ def deactivate_run_enrollment(run_enrollment, change_status):
             CourseRunEnrollment: The deactivated enrollment
         """
         run_enrollment.deactivate_and_save(change_status, no_user=True)
+        try:
+            unenroll_edx_course_run(run_enrollment)
+        except Exception as exc:  # pylint: disable=broad-except
+            self.stdout.write(self.style.ERROR(str(exc)))
         mail_api.send_course_run_unenrollment_email(run_enrollment)
```

Comment:
Should we be sending an email in case there was an error while processing? Say, for whatever reason the unenrollment fails the email is still going out.

Reply:
I just checked what we do when we add enrollments, and it looks like we only send the email if the enrollment also succeeds in edX. I think it makes enough sense to do the same here. Good catch!